guides/common/modules/proc_restoring-from-a-full-backup.adoc

```diff
@@ -43,7 +43,15 @@ endif::[]
 . Choose the appropriate method to install {Project} or {SmartProxy}:
 ** To install {ProjectServer} from a connected network, follow the procedures in {InstallingServerDocURL}[{InstallingServerDocTitle}].
 ifdef::satellite[]
++
+If you intend to use {insights-iop}, enable it.
```

Comment:
From a logical perspective, I would rather say "if you have used Insights, enable it" because you can only restore from backup if you made a backup in the past that used Insights.